neuralmagic/benchmarks/configs/benchmark_throughput.json

```diff
@@ -34,7 +31,8 @@
 				],
 				"dtype": [
 					"auto"
-				]
+				],
+				"use-all-available-gpus_" : []
```

Comment:
same as above - an empty `use-all-available-gpus_` translates to `--use-all-available-gpus_` arg